core/market/decentralized/src/protocol/negotiation/provider.rs

```diff
@@ -233,6 +233,8 @@ impl NegotiationApi {
         public_prefix: &str,
         _local_prefix: &str,
     ) -> Result<(), NegotiationApiInitError> {
+        log::info!("Negotiation (Provider) protocol version: mk1");
```

Comment:
Note that negotiations don't have versioning in gsb binding points. I wanted to introduce them, but it would be incompatible with our current market, so I decided not to introduce breaking changes in this PR.
And the second thing: This is not `DISCOVERY_VERSION`, but Negotiation protocol version. It should be separate const.